src/content/overview/architecture.md

```diff
@@ -1,13 +1,14 @@
 ---
-title: Architecture of a Cloud Native Developer Platform
-description: TBD.
+title: Architecture of a cloud-native developer platform
+linkTitle: Architecture of the platform
+description: An overview of the architecture of our cloud-native developer platform.
```

Comment:
These don't fit together. Do we show our platform? Then replace "our" – the docs should focus on "you"/"customer" and otherwise substitute "our" with "Giant Swarm". Or do we show how "a" platform looks like? I guess no

Also, the description is just a repetition of the title. That's a no-go.

```suggestion
title: Architecture of the Giant Swarm cloud-native developer platform
linkTitle: Architecture of the platform
description: Components, capabilities, supported cloud providers, and the Management API of the Giant Swarm platform.
```